terraso_backend/apps/core/models/users.py

```diff
@@ -114,6 +115,36 @@ def soft_delete_policy_action(self, **kwargs):
     def is_group_manager(self, group_id):
         return self.memberships.managers_only().filter(group__pk=group_id).exists()
 
+    def full_name(self):
+        return f"{self.first_name} {self.last_name}".strip()
```

Comment:
Suggestion: If the structure of the full name can be localized it will be more flexible for other languages in the future, the first_name last_name order is not the same in all languages

Reply:
That's a good point. How do you want to handle it? Is there a module/library for this `make_full_name(first, last, locale)`?

Reply:
@paulschreiber not sure in python, in the web client we are handling this with interpolations so there is a string that can define the order for each locale: https://github.com/techmatters/terraso-web-client/blob/main/src/localization/locales/es-ES.json#L13, so if there are interpolations in the library that you are using that should be enough

Reply:
Great. I've done the equivalent in Python.